app/assets/stylesheets/components/_modal.scss

```diff
@@ -1,27 +1,3 @@
-.modal-backdrop {
-  background-color: rgba(91, 97, 106, 0.95);
-  display: table;
-  height: 100%;
-  left: 0;
-  position: fixed;
-  top: 0;
-  width: 100%;
-  z-index: 1000;
```

Comment:
FYI in testing this, I just noticed a small regression with this change where some content can peak through the modal ([example screenshot](https://user-images.githubusercontent.com/1779930/153940290-63d5386a-2307-4024-9f09-55b4d753d983.png)). I hoped it might be [an easy fix](https://github.com/18F/identity-idp/compare/aduth-modal-overlay-wrapper), but it looks like USWDS adds [some styles](https://github.com/uswds/uswds/blob/c0b0b4d1e3588a748aa503b95dd0c79a3395a9a4/src/stylesheets/components/_modal.scss#L20) we might not be ready for. I might see about just overriding those individual styles, but if it came to it, reverting or backing out changes from #5928 could be a more direct option.

Reply:
Ok! Let me know by Wednesday what option you'd like to move ahead with

Reply:
For posterity: This was fixed in #5949